airqo_monitor/format_data.py

```diff
@@ -110,3 +110,20 @@ def get_and_format_data_for_all_channels(start_time=None, end_time=None):
         all_channels_dict[channel.channel_id] = {'channel': channel, 'data': data}
 
     return all_channels_dict
+
+def get_and_format_heatmap_data_for_all_channels(start_time=None, end_time=None):
+    all_channels_dict = get_and_format_data_for_all_channels(start_time, end_time)
+    geojson_data = {"type": "FeatureCollection", "features": []}
+
+    for channel_info in all_channels_dict.values():
+        if channel_info["channel"].channel_type.name == "airqo":
```

Comment:
you can use the `AIRQO_CHANNEL_TYPE` constant in case this changes

Reply:
done.